### PR TITLE
Use GitHub api to fetch changelog

### PR DESCRIPTION
**Pull request by Nazar1ky**
Better to use api to fetch changelog.

### CODE REVIEW

File: cogs/utils/github.py [hunk 1]
```diff
@@ -13,9 +12,21 @@ def check_for_updates(version):
     return False
 
 def changelog():
-    latest = get_latest()
-    response = requests.get(f"https://github.com/r-liner/discord-commands-bot/releases/tag/v{latest}")
-    soup = BeautifulSoup(response.text, "html.parser")
-    _changelog = soup.find("div", class_="markdown-body my-3").get_text()
+    # Use text media_type: https://docs.github.com/en/rest/using-the-rest-api/getting-started-with-the-rest-api#media-types
```
Comment (liner-exe) [inline cogs/utils/github.py:15]: Remove comment

File: cogs/utils/github.py [hunk 2]
```diff
@@ -13,9 +12,21 @@ def check_for_updates(version):
...
+    headers = {
+        "Accept": "application/vnd.github.v3.text+json",
+    }
+
+    response = requests.get(
+        "https://api.github.com/repos/liner-exe/discord-commands-bot/releases/latest",
+        headers=headers,
+    )
+
+    response.raise_for_status()
```
Comment (liner-exe) [inline cogs/utils/github.py:25]: Remove debug function call

**Review by liner-exe — CHANGES_REQUESTED** [review]

File: cogs/utils/github.py (hunk 2 above)
Comment (Nazar1ky) [reply cogs/utils/github.py:25]: Maybe if status not okay, return nothing ""?

File: cogs/utils/github.py (hunk 1 above)
Comment (Nazar1ky) [reply cogs/utils/github.py:15]: That's reference to official GitHub api not sure why comment should be removed, but okay

File: cogs/utils/github.py (hunk 2 above)
Comment (liner-exe) [reply cogs/utils/github.py:24]: Yep

**Pull request merged**